portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,36 @@
+/* body {
```

Comment:
While commenting code out as you're working is very useful, I think that for most changes which are sent out for review for a shared project, it's generally preferred to remove commented out code. Thanks to versioning in git, we can always get this back if we need it, so it's not really "lost" by removing it.

This has been pretty universal at the companies that I've worked for. The main exception being things that are commented out very temporarily (e.g. with the intention of disabling some behavior to be soon turned back on).

Reply:
Understood I will reduce the number of comments in my code next time.